showcase/app/components/page-components/copy/button/code-fragments/with-masked-input.gts

```diff
@@ -0,0 +1,92 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import type { TemplateOnlyComponent } from '@ember/component/template-only';
+import { eq } from 'ember-truth-helpers';
+
+import {
+  HdsCopyButton,
+  HdsFormMaskedInputBase,
+  HdsFormField,
+  HdsFormMaskedInputField,
+} from '@hashicorp/design-system-components/components';
+
+type variant =
+  | 'masked-input-base'
+  | 'masked-input-base-form-field'
+  | 'masked-input-field';
+
+export const MASKED_INPUT_VARIANTS: variant[] = [
```

Comment:
[suggestion] see comment above about a possible simplification to avoid duplicating the list